.expeditor/config.yml

```diff
@@ -0,0 +1,57 @@
+# Documentation available at https://expeditor.chef.io/docs/getting-started/
+---
+project:
+  alias: chef-semverse
+
+# The name of the product keys for this product (from mixlib-install)
+product_key:
+  - chef-semverse
+
+# Slack channel in Chef Software slack to send notifications about build failures, etc
+slack:
+  notify_channel: chef-semverse-notify
```

Comment:
we could specify chef-ws-notify channel instead. the idea being to have a common notification channel for all gems maintained by this team.

Reply:
Will do the subsequent changes in upcoming PR